TextMate/TextMate2.download.recipe

```diff
@@ -13,29 +13,28 @@ preferences panel.
     <string>com.github.autopkg.download.TextMate2</string>
     <key>Input</key>
     <dict>
+        <key>BASE_URL</key>
```

Comment:
This should be its own PR and we should merge this change into master, and then rebase our py2-to-3 branch on it.

Reply:
Done. TextMate changes now in separate PR #318